src/csi/go.mod

```diff
@@ -1,47 +1,72 @@
 module github.com/poseidonos/pos-csi
 
-go 1.14
+go 1.19
 
 require (
-	github.com/container-storage-interface/spec v1.2.0
-	github.com/golang/protobuf v1.4.2 // indirect
-	github.com/google/uuid v1.1.1
+	github.com/container-storage-interface/spec v1.7.0
+	github.com/google/uuid v1.3.0
 	github.com/kubernetes-csi/csi-lib-utils v0.7.0
-	github.com/onsi/ginkgo v1.11.0 // indirect
-	github.com/onsi/gomega v1.7.0 // indirect
-	github.com/spdk/spdk-csi v0.0.0-20211122155459-a55a004bcf44 // indirect
-	google.golang.org/grpc v1.27.1
-	k8s.io/apimachinery v0.19.3 // indirect
-	k8s.io/client-go v0.19.3 // indirect
+	google.golang.org/grpc v1.52.0
 	k8s.io/klog v1.0.0
-	k8s.io/klog/v2 v2.30.0
-	k8s.io/kubernetes v1.19.3 // indirect
-	k8s.io/mount-utils v0.23.1
-	k8s.io/utils v0.0.0-20210930125809-cb0fa318a74b
+	k8s.io/klog/v2 v2.80.1
+	k8s.io/kubernetes v1.26.1
+	k8s.io/mount-utils v0.26.1
+	k8s.io/utils v0.0.0-20221107191617-1a15be271d1d
 )
 
-replace (
-	// https://github.com/etcd-io/etcd/issues/11563
-	google.golang.org/grpc => google.golang.org/grpc v1.26.0
-	k8s.io/api => k8s.io/api v0.19.3
-	k8s.io/apiextensions-apiserver => k8s.io/apiextensions-apiserver v0.19.3
-	k8s.io/apimachinery => k8s.io/apimachinery v0.19.3
-	k8s.io/apiserver => k8s.io/apiserver v0.19.3
-	k8s.io/cli-runtime => k8s.io/cli-runtime v0.19.3
-	k8s.io/client-go => k8s.io/client-go v0.19.3
-	k8s.io/cloud-provider => k8s.io/cloud-provider v0.19.3
-	k8s.io/cluster-bootstrap => k8s.io/cluster-bootstrap v0.19.3
-	k8s.io/code-generator => k8s.io/code-generator v0.19.3
-	k8s.io/component-base => k8s.io/component-base v0.19.3
-	k8s.io/cri-api => k8s.io/cri-api v0.19.3
-	k8s.io/csi-translation-lib => k8s.io/csi-translation-lib v0.19.3
-	k8s.io/kube-aggregator => k8s.io/kube-aggregator v0.19.3
-	k8s.io/kube-controller-manager => k8s.io/kube-controller-manager v0.19.3
-	k8s.io/kube-proxy => k8s.io/kube-proxy v0.19.3
-	k8s.io/kube-scheduler => k8s.io/kube-scheduler v0.19.3
-	k8s.io/kubectl => k8s.io/kubectl v0.19.3
-	k8s.io/kubelet => k8s.io/kubelet v0.19.3
-	k8s.io/legacy-cloud-providers => k8s.io/legacy-cloud-providers v0.19.3
-	k8s.io/metrics => k8s.io/metrics v0.19.3
-	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.19.3
+require (
```

Comment:
Can you please let me know  why replace is changed to require with //indirect?

Reply:
Replace is used here to use the local module instead of the remote one, which was not good practice.
Indirect is used to avoid downloaded and built dependency in the final build artifact for the Module

Reply:
Thank you for the explanation.